spec/factories/tasks.rb

```diff
@@ -5,5 +5,10 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    trait :done_task do
+      status {2}
+      completion_date {Time.current.yesterday}
```

Comment:
細かいですがブレース（波括弧）の間はスペースを開けた方が見やすいですね。
```suggestion
      completion_date { Time.current.yesterday }
```

Reply:
4251094 ブレースの間に半角スペースを入れて見やすくしました。